src/main/java/com/gl/donate_receive/controller/UserController.java

```diff
@@ -20,9 +21,19 @@ public UserController(UserService userService) {
 		this.userService = userService;
 	}
 
+	@GetMapping
+	public String create(Model model) {
```

Comment:
@GetMapping doesn't align with the method name. Should be something like getCreationForm